wiki/Game_mode/osu!catch/en.md

```diff
@@ -1,78 +1,84 @@
+- Fruits
+- Juice Streams
+- Banana Showers
 
-#### Fruits
+### Fruits
 
 ![Fruits](/wiki/shared/Catch_fruits.jpg "osu!catch fruits")
 
-To catch these fruits, make sure it falls to the plate and not over the plate. Collected fruits will always give 300 points each, increment combo by 1, and treated as 300 in result screen.
+**Fruits** are the simplest object type in osu!catch, all the user is required to do is move the platter under the fruit so it will fall on to the plate rather than over it.
 
-#### Fruit trails
+Collected fruits will always give a `GREAT` judgement, rewarding 300 points each and increment combo by 1. 
 
-![Fruit trails](/wiki/shared/Catch_trails.jpg "osu!catch fruit trails")
+### Juice streams 
 
-A typical fruit trail will contains two fruits (start and end), droplets (slider path), juice drop (slider ticks), and more fruits for each reverse arrow points along with a reversed slider path. Droplet gives 10 points each and treated as 50 in result screen. Juice drop gives 30 points each, increment combo by 1, and treated as 100 in result screen.
+A typical **juice stream** will contains two fruits at its start and end. Between the start and the end there will also be a path made out of small and large droplets, with the large droplets acting as slider ticks. Juice streams, similarly to sliders, can be reversed.
 
-Missing the juice drop will break the combo (as it is a slider tick), but missing the droplet will not break the combo. However, missed droplets will not be shown in the local result screen, but will be shown in the website's leaderboard as *Miss Droplet*, which notably affects accuracy (missed out the droplet, which is a 50) and score (10 points for each droplet missed).
+Small droplets give 10 points each and don't affect combo, while large droplets give 30 points, increment combo by 1 if caught. Unlike small droplets, missing a large droplet resets the combo
 
-#### Bananas
+### Banana showers
 
 ![Bananas](/wiki/shared/Catch_bananas.jpg "osu!catch bananas")
 
-Here, large bananas will rain down and shrink down to catch size as it falls down to the plate (or not). Each collected bananas will give a static score of 1,100 regardless of mod and combo multiplier.
+While a **banana shower** is active, large bananas will pour from the sky and shirk down. Catching bananas (or missing them) doesn't affect combo, but if caught they reward 1,100 score each and regenerate a portion of the players missing health. Each banana will always reward 1,100 score regardless of mod and combo multiplier.
 
-The bananas can be ignored with no health penalty. However, it can cost the extra points and health recovery from collecting the bananas. Notably, [Auto](/wiki/Gameplay/Game_modifier/Auto) mod will go above and beyond to catch *all* of the bananas without fail.
+Notably, [Auto](/wiki/Gameplay/Game_modifier/Auto) mod will go above and beyond to catch *all* of the bananas without fail, even if its humanly impossible.
```

Comment:
it's not just humanly impossible - auto disregards gameplay mechanics and moves the catcher at arbitrary speeds (or i guess it plays on relax)

```suggestion
Notably, [Auto](/wiki/Gameplay/Game_modifier/Auto) mod will go above and beyond to catch *all* of the bananas without fail, even if it is not possible in normal gameplay.
```